p4-16/spec/P4-16-spec.mdk

```diff
@@ -2226,6 +2231,30 @@ header Empty_h { }
 
 Note that an empty header still contains a validity bit.
 
+When a struct is inside of a header, the order of the fields for the purposes 
+of extract and emit calls is the order of the fields as defined in the source code.
```

Comment:
@mbudiu-vmw Both extract and emit are mentioned here.  Is this enough?

@hesingh As a typographic nicety, it might be nice to have them in backquotes so they show up as a different font like other identifiers and keywords in the spec.